Update ESLyric to version 0.5.2

Timmyfox opened the pull request:
Adds compatibility for the new 0.5.2 version of ESLyric (https://github.com/ESLyric/release) which is incompatible with the settings for the earlier 0.3.x version used by Eole currently. **Note that this also means that this update breaks compatibility with earlier versions of ESLyric**.

It is not perfect as the new version has script support disabled by default and requires a setting change, or else the spider monkey panel controlling ESLyric will give an error message. I have amended the readme and installation instructions with these steps.

Also there does not seem to be any setting to control desktop lyrics, hence I commented out those, although it appears that 0.5.2 has them disabled by default so it should not be a problem.

Ottodix commented:
Hi,
I would merge this request, if it wasn't for this error message, which may raise support issues here (because people often misread the installation instructions). I've asked on the github repo of this new ESLyric if it could be set to 1 by default. But well, the guy who coded this disabled the "issues" feature of his repo, so he may not be willing to do this change, we'll see.

So, pending status for the moment

Timmyfox commented:
That's understandable, hopefully they can change it or otherwise offer a more tangible solution.

I'll update if I find any workarounds that might work.

I suppose one option, if they won't change the default, could be to wrap the activexobject call in a try-catch and prompt the user to enable scripts if the setting is set to 0.

That way it will tell the user what to do instead of giving an error, although the panel also won't function as intended.

For example, in the Now Playing tab where it normally says "Lyrics":
![image](https://user-images.githubusercontent.com/1836111/180905195-ec7c373c-153f-47c9-9718-3a643d2c3deb.png)

a1waysbeta commented:
You can wait a little while, the ESL authors will take care of this issue.

Timmyfox commented:
Alright, seems like 0.5.2.1120 resolves this by adding a `enable_script_control` file in the plugin install folder, I've updated to this version and amended the file.

Also removed the readme addition and install instructions prompting the user to change the setting as it now defaults to `1` with the file in place.

Ottodix commented:
Yep, it seems perfect to me, so I merged your pull request. It indeed seems to find lyrics a lot better now.
Thanks for this work!
